cosmos/lcd.go

```diff
@@ -180,10 +180,15 @@ func (lcd *LCD) getAccount() (*auth.BaseAccount, error) {
 		}
 		lcd.acc = auth.NewBaseAccount(accKb.GetAddress(), nil, accKb.GetPubKey(), 0, 0)
 	}
-	localSeq := lcd.acc.GetSequence()
-	if err := lcd.Get("auth/accounts/"+lcd.acc.GetAddress().String(), &lcd.acc); err != nil {
+	var accR *auth.BaseAccount
+	if err := lcd.Get("auth/accounts/"+lcd.acc.GetAddress().String(), &accR); err != nil {
 		return nil, err
 	}
+	if accR.Address.Empty() {
+		return nil, fmt.Errorf("account %q doesn't exist", lcd.acc.GetAddress().String())
+	}
+	localSeq := lcd.acc.GetSequence()
```

Comment:
Why move this one? the goal was to have the local sequence and compare it to the sequence on the chain.
Here the local sequence becomes totally useless and so the condition 2 lines bellow `localSeq > lcd.acc.GetSequence()`.
Either this needs to be put back before the fetch or the whole local sequence need to be deleted

Reply:
Look carefully, the get store the fetched account in variable `accR`.
If the sequence system was broken, the e2e test that executes 10 executions in the same block will not work.

Reply:
Hum yes you're right I missed the part that it was replaced... The code is a bit weird. Maybe something like that would be less confusing.
```go

	// replace seq if sup
	if localSeq > accR.GetSequence() {
		accR.SetSequence(localSeq)
	}
	lcd.acc = accR
```